Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.h

```diff
@@ -259,7 +259,7 @@ namespace AssetProcessor
         void JobRemoved(AzToolsFramework::AssetSystem::JobInfo jobInfo);
 
         void JobComplete(JobEntry jobEntry, AzToolsFramework::AssetSystem::JobStatus status);
-        void JobProcessDurationChanged(JobEntry jobEntry, QTime duration);
+        void JobProcessDurationChanged(JobEntry jobEntry, int duration);
```

Comment:
"duration" is now "durationMs" for microseconds?

Reply:
That is a good point.

Reply:
resolved

Reply:
resolved